src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -3880,7 +3880,7 @@ private void LT_Handler(Instruction instruction)
             {
                 double value1 = opdata2.IsDouble ? opdata2.DoubleValue: opdata2.IntegerValue;
                 double value2 = opdata1.IsDouble ? opdata1.DoubleValue: opdata1.IntegerValue;
-                opdata2 = StackValue.BuildBoolean(MathUtils.IsLessThan(value1, value2));
+                opdata2 = StackValue.BuildBoolean(value1 < value2);
```

Comment:
I'm concerned that this loses the toleranced comparison.  If value1 was less than value2 but within the floating point tolerance, then this will change from false to true.  Example: 0.1 and 0.100000001 would be false with tolerance, and true without.  If

Reply:
Okay, just got to read the comments on the PR itself, and see why this is being done.